packages/pluggableWidgets/line-chart-native/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "line-chart-native",
   "widgetName": "LineChart",
-  "version": "1.0.0",
```

Comment:
I think you need a rebase

Reply:
Indeed, locally I have Isa's latest styling changes as a base.

Reply:
Yeap, soon as we merge his PR you can rebase 😃